playbooks/README.md

```diff
@@ -20,6 +20,7 @@ practice should be incorporated, submit a PR!
 <!-- start_toc -->
 | Doc | Overview |
 |--|--|
+| [Request for Comments [RFC]](/playbooks/rfcs.md) | How do we make a Request for Comments? |
```

Comment:
I think this link already exists below with the title "How to create an RFC at Artsy".